runtime/compiler/ilgen/Walker.cpp

```diff
@@ -5465,13 +5465,19 @@ TR_J9ByteCodeIlGenerator::loadFromCP(TR::DataType type, int32_t cpIndex)
             // Use aconst for null object
             if (!isCondyPrimitive && !isCondyUnresolved)
                {
-               TR::VMAccessCriticalSection condyCriticalSection(comp()->fej9());
-               uintptrj_t* objLocation = (uintptrj_t*)_methodSymbol->getResolvedMethod()->dynamicConstant(cpIndex);
-               uintptrj_t obj = *objLocation;
-               if (obj == 0)
+               TR::VMAccessCriticalSection condyCriticalSection(comp()->fej9(),
```

Comment:
This is a change in behavior; previously the compilation thread acquired VM access and potentially waited to do so. With the new code the compilation thread attempts to acquire VM access, but if it cannot get it right away, it bails out.
I would like some confirmation from the optimizer team that this new behavior is ok (may have performance implications) and that there is a safe path if we don't wait for VM access.

Reply:
Shouldn't change it to `tryToAcquireVMAccess` since we're not dereferencing the `obj`. Will update the code.